harvester_e2e_tests/fixtures/network.py

```diff
@@ -36,12 +36,25 @@ def enable_vlan(request, admin_session, harvester_api_endpoints):
         vlan_json['config'] = {}
     if 'defaultPhysicalNIC' not in vlan_json['config']:
         vlan_json['config']['defaultPhysicalNIC'] = None
-    vlan_json['enable'] = True
-    vlan_json['config']['defaultPhysicalNIC'] = vlan_nic
-    utils.poll_for_update_resource(request, admin_session,
-                                   harvester_api_endpoints.update_vlan,
-                                   vlan_json,
-                                   harvester_api_endpoints.get_vlan)
+
+    for item in request.session.items:
+        if item.get_closest_marker('terraform') is not None:
```

Comment:
No need to do the for loop here. I would put the marker lookup logic into utils.py.

Reply:
fixed